### PR TITLE
Template issue to request merge rights

### PR DESCRIPTION
**Pull request by marcusburghardt**
#### Description:

It was agreed in some criteria about rights grant in the project.
- https://github.com/ComplianceAsCode/content/blob/master/docs/manual/developer/01_introduction.md#governance 

The initial proposal mentioned that requests should be done via email or Gitter.
This PR updates this part by centralizing these requests in the Project Issues.
This PR also creates an issue template for Merge Rights request, in order to make these requests even easier.

#### Rationale:

It is more transparent to centralize these requests in the project itself.
It also permits an easier tracking of grants and ensure a standard.
Once we have the request filed in a form of an issue, maintainers can easier manifest their approvals or concerns via comments.

This PR is not changing the already defined rules, but only improving how they are supported.

### CODE REVIEW

**Comment by jan-cerny** [comment]
I will repeat myself that it's great that we have the process but we need to actively offer this opportunity to people.

**Comment by marcusburghardt** [comment]
> I will repeat myself that it's great that we have the process but we need to actively offer this opportunity to people.

Yes, once we have a organized way to support this process, it is much easier to practice it.

File: .github/ISSUE_TEMPLATE/request_merge_rights.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+about: Request merge rights to help on the project maintenance
+labels: governance
+---
+#### Github ID:
+
+* Include the URL for the github profile.
+
+#### Reason for having merge rights:
+
+* Give a clear and objective context about why do you need merge rights.
```
[the remark this answers is not in the document]
Comment (marcusburghardt) [reply .github/ISSUE_TEMPLATE/request_merge_rights.md]: Nice @matejak . Please, take a look in the last commit.

**Review by matejak — APPROVED** [review]
LGTM, we can always improve the wording according to the feedback if there is a need. Good job!

**Pull request merged**